Taskfile.yml

```diff
@@ -106,7 +106,7 @@ tasks:
         msg: "A value for IMAGE must be provided"
     cmds:
       - cat playtron-os_kickstart.cfg.template | envsubst '${REGISTRY} ${REGISTRY_TOKEN} ${PROJECT} ${IMAGE} ${TAG}' > playtron-os_kickstart.cfg
-      - sudo setenforce 0
+      - if [ -f /usr/sbin/setenforce ]; then sudo setenforce 0; fi
```

Comment:
On my Fedora desktop, this is the correct path, but on GameOS, the setenforce binary is in /usr/bin.

While unlikely to be a problem, it is probably better not to assume the binary file's location.

I recommend using `command -v` instead to check for the existence of a command.

Reply:
Updated.